docs/modules/ROOT/pages/commands-list.adoc

```diff
@@ -0,0 +1,176 @@
+= Comprehensive List of Commands
+
+Below is the complete list of supported commands:
+
+* xref:commands/a.adoc[@a]
```

Comment:
This page is kind of boring and ugly.

Commands could be on a table like the ones mrdocs generates where the first column has the command and the second column has a brief description.

To make it less repetitive, commands can be put into subsections categories. The basic categories are block and inline commands. Then there's subcategories based on what the command does.

Each section can have a small paragraph explaining the table to come.

The command text should be as wrapped in "`"s.

Unsupported commands can be in red without a page so that users and developers can keep track of them.

Unimplemented commands are unsupported commands.